logging/log.go

```diff
@@ -1,6 +1,6 @@
 package logging
 
-type LoggerFactory func(ctx ...interface{}) Logger
+type LoggerFactory func(loggerName string, ctx ...interface{}) Logger
```

Comment:
this makes it clear that first element is always logger name whereas the remaining go to context (that's how it is implemented in Grafana)